rules/react.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   'react-hooks/exhaustive-deps': 'error', // Checks effect dependencies
   'react-hooks/rules-of-hooks': 'error', // Checks rules of Hooks
+  'react/jsx-key': 'warn', // Checks required key prop in JSX exist
```

Comment:
주기적으로 TODO 검색하고 있고 또 IDE에서 알람을 받을 수도 있어서요,
주석을 TODO로 달아주시면 더 놓치지 않을 수 있을 것 같습니다.

```suggestion
  'react/jsx-key': 'warn',  // TODO (@axel): Checks required key prop in JSX exist
```

Reply:
반영했습니다!